src/discrete-layer/components/map-container/poi.dialog.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useCallback, useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useFormik } from 'formik';
+import { isEmpty } from 'lodash';
+import * as Yup from 'yup';
+import { Button, Dialog, DialogContent, DialogTitle, IconButton, TextField } from '@map-colonies/react-core';
+import { Box } from '@map-colonies/react-components';
+import { ValidationsError } from '../../../common/components/error/validations.error-presentor';
+import { FieldLabelComponent } from '../../../common/components/form/field-label';
+
+import './poi.dialog.css';
+
+const NONE = 0;
+
+export interface IPOI {
+  lon: number;
+  lat: number;
+}
+
+interface PoiDialogProps {
+  isOpen: boolean;
+  onSetOpen: (open: boolean) => void;
+  onPoiUpdate: (longitude: number, latitude: number) => void;
+}
+
+export const PoiDialog: React.FC<PoiDialogProps> = ({ isOpen, onSetOpen, onPoiUpdate }) => {
+  const intl = useIntl();
+  
+  const closeDialog = useCallback(
+    () => {
+      onSetOpen(false);
+    },
+    [onSetOpen]
+  );
+
+  const [poi] = useState<IPOI>({ lon: 0, lat: 0 });
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const yupSchema: Record<string, any> = {};
+  Object.keys(poi).forEach(fieldName => {
+    const fieldLabel = `general.dialog-field.${fieldName}.label`;
+    yupSchema[fieldName] = Yup
+    .number()
+    .required(
+      intl.formatMessage(
+        { id: 'validation-general.number' },
+        { fieldName: `<strong>${intl.formatMessage({ id: fieldLabel })}</strong>` }
```

Comment:
extract to function and use it in all <strong> places

Reply:
Done